### DISCUSSION TITLE
[Bug]: I have a problem with my wallet since my funds from the migration from TRINITY Wallet to Firefly Wallet never arrived, I do not have access to them

### DISCUSSION THREAD

**Discussion by MidasREDG79** [iotaledger/firefly#2747] [Help & Support]
### Issue description

Hello, I have a problem with my wallet since my funds from the migration from TRINITY Wallet to Firefly Wallet never arrived, I do not have access to them, how could you please solve them?
Also when exporting my security seed it cannot be seen.
I appreciate help me.

Att. Robert Eduardo Duarte Gonzalez

### Expected behaviour

I thought that at the time of migration my balance would arrive, but it was not like that

### Actual behaviour

says your funds will be available shortly and they are not

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

1. I go into my wallet
2. I give you unlock funds
3. funds do not appear

### Diagnostics

```shell
Firefly -1.3.3
Idioma: es-LA
Moneda: USD
Selección de nodo: Automático
Plataforma: win32
Versión de la plataforma: 10.0.19043
Arquitectura de la plataforma: x64
Recuento de CPU: 2
Memoria total: 3983.9 MB
Memoria libre: 643.3 MB
```

### Errors

```shell
EMPTY
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MiKeF05** [comment]
Hello. Can you try to create a new profile and restore it with your 24 words?

**Comment by SebKrom** [comment]
I have exactly the same problem.

**Comment by SebKrom** [comment]
I tried it with 81 sign and with ledger integration but both failed and my funds were not shown.

**Comment by MiKeF05** [comment]
@SebKrom  I guess you are refering to something else. You mean that your balance is not found in the migration process?

You need to know where your funds are located. Did you use a ledger or normal profile with 81 chars? If you have Trinity installed you can login and check your funds.

**Comment by SebKrom** [comment]
Yes but isnt that the same Problem with other words? 

I'm now afraid that something could happened to my funds. I can no longer access it in the old Trinity wallet either, because it no longer works.

**Comment by MiKeF05** [comment]
No in the issue above the migration was completed but there is a display error.

What's the issue with Trinity? You only can't login?
Try the following, update Trinity to the latest version. Before you open Trinity disable your internet connection. Then you should be able to login and check where your funds are located.

**Comment by SebKrom** [comment]
This way i can see my Funds in the Trinity wallet but how can i migrate? 

With ledger it does not work. Firefly is just searching for funds for Hours but cant find. 

Kind regards 

Sebastian

**Comment by MiKeF05** [comment]
Your funds are located to the ledger? If you go to view seed in the account in Trinity do you see a 81 char seed? Or does it say the seed phrase can't be exported or displayed?

**Comment by SebKrom** [comment]
Oh, ist was the Index. I had Index one. Now it works. Thank you for Support.

**Comment by SebKrom** [comment]
Now i have another question. The migration process was successfull and i saw my funds on the Dashboard, but when i closed firefly and started it again it asked me to migrate it asked me to create a new profil. Like i never migrated.

What can i do?

Kind regards 
Sebastian

**Comment by MiKeF05** [comment]
You can create a new profile and recover your wallet using the option hardware wallet and Firefly ledger backup.
